lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -47,6 +58,36 @@ def release_languages(self, languages):
             enabled=True
         ).save()
 
+    def create_user(self):
+        """
+        Creates the user log in
+        """
+        # Create one user and save it to the database
+
+        self.user = UserFactory.build(username='test', email=self.email)
+        self.user.set_password(self.pwd)
+        self.user.save()
+
+        # Create a registration for the user
+        RegistrationFactory(user=self.user)
+
+        # Create a profile for the user
+        UserProfileFactory(user=self.user)
```

Comment:
This was part of the original code, I pulled the common code into this new method.  Thank you for the heads up I will remove this line then.